app/views/users/registrations/edit.html.slim

```diff
@@ -51,6 +51,17 @@
               autocomplete: 'new-password',
               class: 'form-control'
 
+    .form-group.field-element
+      = f.label :openai_api_key, t('users.show.openai_api_key'), class: 'form-label'
+      = f.text_field :openai_api_key,
+              required: false,
+              placeholder: t('users.show.openai_api_key'),
+              autocomplete: 'off',
+              class: 'form-control'
+      small.form-text.text-body-secondary
+        = t('devise.registrations.edit.api_key_usage')
+        = link_to 'OpenAI API website', 'https://platform.openai.com/api-keys', target: '_blank', rel: 'noopener noreferrer'
```

Comment:
where in config/locales? I thought I used the correct destination as it is in the `registrations.edit` and is used previously for other translations.  Could you be more specific so I can edit it correctly? :)

Reply:
The destination is correct. I am just saying: Please move the string 'OpenAI API website' from the view (where it currently lives) to  the same place where the previous text is stored (currently `devise.registrations.edit.api_key_usage`).

Apropos talking about the previous text: The location `devise.registrations.edit.api_key_usage` doesn't seem perfect, since everything in the `devise` namespace is externally managed (through the Devise gem for authentication). What about `users.registrations.edit.api_key_usage`? In the view, this might be as simple as `.api_key_usage`. (And, finally, with the `openai_` prefix, resulting in: `.openai_api_key_usage`?)